index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile-main</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+   <div>
+     <div class="logo">
+       <h3>My Logo</h3>
+       <img class="img" src="Union.png" alt="logo of vector">
```

Comment:
The container of the section should be a header tag. Please replace div with a header and use flex box to align content.